pkg/storage/unified/apistore/history.go

```diff
@@ -101,3 +103,12 @@ func (r *historyREST) Connect(ctx context.Context, uid string, opts runtime.Obje
 		responder.Object(http.StatusOK, list)
 	}), nil
 }
+
+// TODO: This is a temporary copy of the function from pkg/services/apiserver/endpoints/request/namespace.go
+func NamespaceInfoFrom(ctx context.Context, requireOrgID bool) (claims.NamespaceInfo, error) {
```

Comment:
i was thinking that this could be moved in a follow-up PR. this unblocks @onprem for now